selfdrive/ui/qt/offroad/developer_panel.cc

```diff
@@ -43,8 +50,33 @@ void DeveloperPanel::updateToggles(bool _offroad) {
     AlignedBuffer aligned_buf;
     capnp::FlatArrayMessageReader cmsg(aligned_buf.align(cp_bytes.data(), cp_bytes.size()));
     cereal::CarParams::Reader CP = cmsg.getRoot<cereal::CarParams>();
+
+    const QString alpha_long_description = QString("<b>%1</b><br><br>%2")
+      .arg(tr("WARNING: openpilot longitudinal control is in alpha for this car and will disable Automatic Emergency Braking (AEB)."))
+      .arg(tr("On this car, openpilot defaults to the car's built-in ACC instead of openpilot's longitudinal control. "
+              "Enable this to switch to openpilot longitudinal control. Enabling Experimental mode is recommended when enabling openpilot longitudinal control alpha."));
+    alphaLongToggle->setDescription("<b>" + alpha_long_description + "</b>");
+
+    if (!CP.getExperimentalLongitudinalAvailable() && !CP.getOpenpilotLongitudinalControl()) {
+      params.remove("ExperimentalLongitudinalEnabled");
+      alphaLongToggle->setEnabled(false);
+      alphaLongToggle->setDescription("<b>" + tr("openpilot longitudinal control may come in a future update.") + "</b>");
+    } else {
+      if (is_release) {
+        params.remove("ExperimentalLongitudinalEnabled");
+        alphaLongToggle->setEnabled(false);
+        alphaLongToggle->setDescription("<b>" + tr("An alpha version of openpilot longitudinal control can be tested, along with Experimental mode, on non-release branches.") + "</b>");
+      }
+    }
```

Comment:
just match the current behavior of hiding this toggle, and let's not add any new language in this PR

Reply:
done